indico/migrations/versions/20200728_1842_d89abffb0f63_add_menu_entries_registered_only.py

```diff
@@ -0,0 +1,26 @@
+"""Add menu_entries.registered_only column.
+
+Revision ID: d89abffb0f63
+Revises: 05f227f4b938
```

Comment:
Could you update the alembic revision so it revises `4f0b4dd412b5`? This is a small fix I added earlier today. You'll also need to rename the file to update the date to today so it shows up at the end of the file list in the versions folder.